packages/flutter/lib/src/material/chip_theme.dart

```diff
@@ -0,0 +1,425 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+import 'colors.dart';
+import 'theme.dart';
+import 'theme_data.dart';
+
+/// Applies a chip theme to descendant [RawChip]-based widgets.
```

Comment:
it would be helpful to say "[RawChip]-based widgets, like [ActionChip], [InputChip] ...."